operations/customers.md

```diff
@@ -134,6 +134,7 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
             "Expiration": "2020-01-01",
             "Issuance": "2016-01-01",
             "IssuingCountryCode": "CZ"
+            "IssuingCity": ""
```

Comment:
Can you supply an example of real data please

Reply:
<img width="554" alt="Iss
<img width="544" alt="Issuingcity2" src="https://github.com/MewsSystems/gitbook-connector-api/assets/134269022/56d4d7e9-5004-4f57-8f66-7c012f66d078">
uingcity" src="https://github.com/MewsSystems/gitbook-connector-api/assets/134269022/f3b2ad05-6191-4ee2-a976-9dac2f737c97">